jit/codegen_x86.cpp

```diff
@@ -4733,9 +4733,16 @@ LOWFUNC(NONE,NONE,2,raw_ftan_rr,(FW d, FR s))
 }
 LENDFUNC(NONE,NONE,2,raw_ftan_rr,(FW d, FR s))
 
+#ifdef CPU_x86_64
+#define REX64 emit_byte(0x48);
```

Comment:
@rofl0r It if fixes an actual problem for you, you are welcome to create a pull request with a commit.